tests/snapshots/test_api_routes/test_advanced_chat_flow/advanced_chat_flow_response.json

```diff
@@ -64,5 +64,5 @@
         ],
         "followup_questions": null
     },
-    "session_state": null
+    "sessionState": null
```

Comment:
Oh, good catch, it's camel cased in the Chat Protocol SDK. I've sent a PR to update spec-
https://github.com/microsoft/ai-chat-protocol/pull/68

Reply:
Yeah caught it while testing.